java-common-artifacts/src/main/java/com/fullcontact/apilib/models/Response/MatchResponse.java

```diff
@@ -7,16 +7,13 @@
 @AllArgsConstructor(access = AccessLevel.PROTECTED)
 @NoArgsConstructor
 public class MatchResponse extends FCResponse {
-  private Boolean city,
-      region,
-      country,
-      continent,
-      postalCode,
-      familyName,
-      givenName,
-      phone,
-      email,
-      maid,
-      social,
-      nonId;
+  private String city;
+  private String region;
+  private String country;
+  private String postalCode;
+  private String familyName;
+  private String givenName;
+  private String phone;
+  private String email;
+  private Double risk;
```

Comment:
This all matches bouncer's MatchResponse, though bouncer has several more fields available now.